app/graphql/mutations/sign_in_mutation.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'ostruct'
+
+module Mutations
+  # GraphQL mutation to sign in a user
+  class SignInMutation < AbstractDeviseMutation
+    # Note, that this does not directly use Devise controller actions to
+    # perform the mutation.  Devise's corresponding controller action,
+    # which would normally be called, performs various things we don't need for
+    # an API only application (including setting flash messages, rendering the
+    # response, bypassing the signin). This code mirrors what is left of the
+    # that action after stripping the rest away.
+    # Make sure that this code is updated if Devise adds relevant code to the
+    # controller action.
+    #
+    # For reference, see the
+    # V2::Users::SessionsController#create and
+    # https://github.com/plataformatec/devise/blob/7a44233fb9439e7cc4d1503b14f02a1d9f6da7b9/app/controllers/devise/sessions_controller.rb#L16-L22
+    def call(_root, arguments, context)
+      setup_devise(context)
+      transform_params_for_devise(arguments)
+      user = authenticate
```

Comment:
No, because this does the actual authentication for signing in. We don't have the `current_user` at this point.  It is generated inside this method.